README.md

```diff
@@ -1 +1 @@
-# Safechain-agent
+# sc-agent
```

Comment:
README.md could use some more documentation about:
- What sc-agent is?
- How it can be setup?
- ...